plugins/oauth/web_client/stylesheets/oauthLoginView.styl

```diff
@@ -101,3 +101,16 @@
 
   .g-oauth-button-icon
     border-right-color darken($brandColor, 30%)
+
+.g-oauth-button-box
+  $brandColor = #205081
```

Comment:
Is this their official color? It looks different from the one I see predominantly on their website.